docs/source.rst

```diff
@@ -100,14 +100,11 @@ Once you have generated a codename and put it somewhere safe, click
 
 |Memorizing your codename|
 
-You will next be brought to the submission interface, where you may
-upload a document, enter a message to send to journalists, or both. You
-can only submit one document at a time, so you may want to combine
-several files into a zip archive if necessary. The maximum submission
-size is currently 500MB. If the files you wish to upload are over that
-limit, we recommend that you send a message to the journalist explaining
-this, so that they can set up another method for transferring the
-documents.
+You will next be brought to the submission interface, where you may upload one
```

Comment:
Once the comment below is addressed, you can convert "you may upload one or more documents" -> can become "you may upload up to 10 documents"